### PR TITLE
Simplify Config

### PR DESCRIPTION
**Pull request by utkuufuk**
Regarding `go.mod` and `go.sum`, I just clicked "Upgrade dependency" on some of the entries via VS Code.

Also another thing I wanted to do was to make logging consistent, i.e. use either `log` or `fmt.Println` but then thought it would be too much for one PR 😄

### CODE REVIEW

File: internal/config/config.go [hunk 1]
```diff
@@ -19,19 +24,59 @@ type Cfg struct {
...
+	}
+
+	return &Cfg{
+		EncryptionKey: string(pwd),
+		MountPoint:    getMountPoint(mntDir),
+		Dropbox:       &DropboxCredentials{dbxToken},
+	}, nil
+}
 
-	return !os.IsNotExist(err)
```
Comment (utkuufuk) [inline internal/config/config.go:31]: @paddlesteamer I'm getting rid of this line and assuming that the dir doesn't exist if I get **any** kind of error. It makes more sense to me this way, but feel free to revert this if I'm missing something.

Comment (paddlesteamer) [reply internal/config/config.go:31]: You're right, actually, this was a buggy behavior before. Now it's fixed.

**Comment by paddlesteamer** [comment]
I think the code is better with these changes. About `log` and `fmt` I want to separate `stderr` and `stdout`. If `log` provides this functionality, let's use `log`. I also want to add log levels. For example, every print to stdout in `fs.go` should be printed with log level 'debug'

**Comment by paddlesteamer** [comment]
So let's merge this. `Log` and `fmt` is another PR's issue.

**Comment by utkuufuk** [comment]
> I think the code is better with these changes. 

@paddlesteamer Then there's one last thing I want to do about the config package, will ping you again when it's ready.

> About `log` and `fmt` I want to separate `stderr` and `stdout`. If `log` provides this functionality, let's use `log`. I also want to add log levels. For example, every print to stdout in `fs.go` should be printed with log level 'debug'

Sounds like a good candidate for a separate issue (if one doesn't already exist).

**Pull request merged**